addons/io_scene_gltf2/blender/exp/animation/gltf2_blender_gather_action.py

```diff
@@ -426,6 +426,9 @@ def gather_action_animations(  obj_uuid: int,
             link_samplers(animation, export_settings)
             animations.append(animation)
 
+            # export user extensions for all types of animation
+            export_user_extensions('gather_animation_hook', export_settings, animation, blender_action, blender_object)
```

Comment:
There is already a hook with the same name

Reply:
Yes this was a suggestion to encounter exported animations extension problem